client/src/hooks/useMetadataOnly.js

```diff
@@ -0,0 +1,19 @@
+import { filterWhere } from 'helpers/filterWhere'
+
+// TODO: Implement the logic to handle multiple metadata files (i.e. project and portal-wide) later
+export const useMetadataOnly = (computedFiles) => {
+  const metadataComputedFile = filterWhere(computedFiles, {
+    metadata_only: true
+  })
+  // Check the availability of metadata_only file
+  const isMetadataOnlyAvailable = metadataComputedFile.length > 0
+
+  // NOTE: The current logic only supports the project level metadata file,
+  // thus the first item is returned
+  return {
+    metadataComputedFile: metadataComputedFile[0],
```

Comment:
```suggestion
  const isMetadataOnlyAvailable = metadataComputedFiles.length > 0

  // NOTE: The current logic only supports the project level metadata file,
  // thus the first item is returned
  return {
    metadataComputedFile: metadataComputedFiles[0],
```

And then use the new name

Reply:
See the commit at 06be077ebea7c8e7c84ac5e5d90f71a377cac85a.